### PR TITLE
Feat: Added Login and Signup 

### LINKED ISSUE #9: Todo: Setup JWT Auth From Mockbee
Todo: Setting up JWT authentication via Mockbee.
Setup Login and Signup Pages.

### PR DESCRIPTION
**Pull request by coldpigli**
image

image

### CODE REVIEW

File: src/components/TextField.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const TextField = ({label, name, placeholder, value, handleChange, type}) => {
```
Comment (nishasen) [inline src/components/TextField.jsx:3]: same name as attribute names may show warnings and cause problems sometimes, you can customise it differently here. eg. , label -> inputLabel

File: src/configs/RoutingPath.jsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ const RoutingPath = () => {
         <Route path="/Cart" element={<Cart/>}/>
         <Route path="/Login" element={<Login/>}/>
         <Route path="/Signup" element={<Signup/>}/>
-        <Route path="Mockman" element={<MockMan/>}/>
+        <Route path="/Mockman" element={<MockMan/>}/>
         <Route path="*" element={<ErrorPage/>}/>
```
Comment (nishasen) [inline src/configs/RoutingPath.jsx:15]: I like how you are making error page.

File: src/pages/Login.jsx [hunk 3]
```diff
@@ -1,6 +1,90 @@
...
+      setUserDetails({
+        cartList: cart,
+        wishList: wishlist,
+        firstName: firstName
+      });
+      localStorage.setItem("userToken",data.encodedToken)
+      navigate("/products")
+    }
+      else{
+        if(response.status=404)
```
Comment (nishasen) [inline src/pages/Login.jsx]: here you are not checking the status, you are assigning 404 to status?

**Review by nishasen — COMMENTED** [review]
Learnt some new ways from your code, also I have added some comments please look into it.

File: src/pages/Login.jsx (hunk 3 above)
Comment (coldpigli) [reply src/pages/Login.jsx]: Ohh. Thank you so much for pointing this out.
I dont know as to how I missed it. Will correct it :D

File: src/index.js [hunk 4]
```diff
@@ -3,17 +3,21 @@ import ReactDOM from "react-dom";
...
 ReactDOM.render(
   <React.StrictMode>
     <Router>
+    <AuthProvider>
+    <ProductProvider>
     <FilterProvider>
       <App />
     </FilterProvider>
+    </ProductProvider>
+    </AuthProvider>
```
Comment (rahulcs754) [inline src/index.js:20]: you can create a router folder and import that one component to her that looks good.

**Review by rahulcs754 — APPROVED** [review]
looks good with perfect design 
just small suggestion about router

**Pull request merged**